src/main/webapp/app/course/plagiarism-cases/instructor-view/plagiarism-cases-instructor-view.component.html

```diff
@@ -20,6 +20,13 @@ <h5 class="mb-0">
                                     <fa-icon [icon]="getIcon(exercise.type)" />
                                 }
                                 {{ exercise.title }}
+                                <small
+                                    ><a
```

Comment:
I think this should be in one line `<samll>` . It's not necessarily wrong, but very unconventional html styling